test/jdk/ProblemList.txt

```diff
@@ -539,12 +539,16 @@ java/net/ServerSocket/AcceptInheritHandle.java                  8211854 aix-ppc6
 
 # jdk_nio
 
-java/nio/channels/DatagramChannel/Unref.java                    8233437 generic-all
+java/nio/channels/AsyncCloseAndInterrupt.java                   8305596 generic-all
+
+java/nio/channels/FileChannel/Transfer.java                     8305596 generic-all
 
 java/nio/channels/AsynchronousSocketChannel/StressLoopback.java 8211851 aix-ppc64
 
 java/nio/channels/DatagramChannel/ManySourcesAndTargets.java    8264385 macosx-aarch64
 
+java/nio/channels/DatagramChannel/Unref.java                    8233437 generic-all
```

Comment:
Is this second change intended? Otherwise looks good.

Reply:
Obsessively alphabetizing.

Reply:
Doh! Sorry for the noise!